docs/build/credentials.md

```diff
@@ -30,8 +30,95 @@ authenticate with your app (e.g., `username`, `api_key`).
 If your app is not listed in the Adaptors section, then inspect your app's API
 documentation to see what is required for "authentication". You can then create
 a `Raw JSON` Credential in OpenFn to define whatever credential inputs are
-require (e.g., `{"api_key": "ADD-your-let", "url": "add-url"}`).
+required. E.g.:
 
-Some systems (Salesforce, OpenMRS, DHIS2) require an instanceUrl, host, or
-ApiUrl. Leave off the final "/" in these Urls: `https://login.salesforce.com` or
-`http://demo.openmrs.org/openmrs` or `https://play.dhis2.org`.
+```json
```

Comment:
The naming convention for adaptor configuration-schema is `camelCase`
```json
{
  "apiKey": "someSecretKey",
  "baseUrl": "https://example.com/api/v2"
}
```

Reply:
addressed in next commit